src/claims/claims.controller.ts

```diff
@@ -0,0 +1,70 @@
+import { Body, Controller, HttpStatus, Post } from '@nestjs/common';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+
+import { ClaimsService } from '~/claims/claims.service';
+import { ModifyClaimsDto } from '~/claims/dto/modify-claims.dto';
+import { ApiTransactionFailedResponse, ApiTransactionResponse } from '~/common/decorators/swagger';
+import { handleServiceResult, TransactionResponseModel } from '~/common/utils';
+import { PolymeshLogger } from '~/logger/polymesh-logger.service';
+
+@ApiTags('claims')
+@Controller('claims')
+export class ClaimsController {
+  constructor(
+    private readonly claimsService: ClaimsService,
+    private readonly logger: PolymeshLogger
+  ) {
+    logger.setContext(ClaimsController.name);
+  }
+
+  @ApiOperation({
+    summary: 'Add Claims targeting an Identity',
+    description: 'This endpoint will add Claims to an Identity',
+  })
+  @ApiTransactionResponse({
+    description: 'Transaction response',
```

Comment:
I think the type needs to be specified so the documentation gets filled in.
```suggestion
    description: 'Transaction response',
     type: TransactionQueueModel,
```